samples/download_view_image.py

```diff
@@ -46,16 +48,29 @@ def main():
     server.version = 2.5
 
     with server.auth.sign_in(tableau_auth):
-        # Step 2: Query for the view that we want an image of
         req_option = TSC.RequestOptions()
+
+        # Step 2: Query for the workbook that contains the view
         req_option.filter.add(TSC.Filter(TSC.RequestOptions.Field.Name,
-                                         TSC.RequestOptions.Operator.Equals, args.view_name))
-        all_views, pagination_item = server.views.get(req_option)
-        if not all_views:
+                                         TSC.RequestOptions.Operator.Equals, args.workbook_name))
+        all_workbooks, pagination_item = server.workbooks.get(req_option)
+
+        if not all_workbooks:
+            raise LookupError("Workbook with the specificed name was not found.")
+        workbook_item = all_workbooks[0]
+
+        server.workbooks.populate_views(workbook_item)
+        view_item = None
+
+        # Step 3: Find the view we want
+        for v in workbook_item.views:
+            if v.name == args.view_name:
```

Comment:
Just a note, this will be case sensitive (and probably should be, Server is as well) -- but maybe add a comment to note that?